spec/requests/api/v1/spots/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'GET api/v1/spots', type: :request do
+  let!(:user)           { create(:user) }
+  let!(:topic)          { create(:topic) }
```

Comment:
I think that the variable `topic` isn't necessary since you don't need a particular topic.